mapper/tedge_mapper/src/mapper.rs

```diff
@@ -1,46 +1,47 @@
-use c8y_translator_lib::json::CumulocityJsonError;
-use mqtt_client::MqttClient;
+use crate::error::{ConversionError, MapperError};
+use mqtt_client::{MqttClient, Topic};
 use tokio::task::JoinHandle;
 use tracing::{debug, error, instrument};
 
-pub const IN_TOPIC: &str = "tedge/measurements";
-pub const C8Y_TOPIC_C8Y_JSON: &str = "c8y/measurement/measurements/create";
-pub const ERRORS_TOPIC: &str = "tedge/errors";
-
 #[derive(Debug)]
+pub struct MapperConfig {
+    pub in_topic: Topic,
+    pub out_topic: Topic,
+    pub errors_topic: Topic,
+}
```

Comment:
I would add an `app_name` property to the `MapperConfig` struct.

* This app name is using as the MQTT session id and as such is a key point of the MQTT config of a mapper.
* Doing so might improve the main.

Reply:
`instrument(debug_span!(APP_NAME))` requires constant value...

Reply:
I talked with @mneumann, then we concluded that we can't do such way because of `debug_span!` requires a constant value.